ql/src/java/org/apache/hadoop/hive/ql/Driver.java

```diff
@@ -905,6 +905,11 @@ public QueryDisplay getQueryDisplay() {
     return driverContext.getQueryDisplay();
   }
 
+  @VisibleForTesting
+  DriverContext getDriverContext() {
```

Comment:
no need to expose context in test. if you need queryPlan, you can simply call:
````
driver.getPlan()
````

Reply:
Removed. Done.